chapters/classes.tex

```diff
@@ -1219,13 +1211,14 @@ \section{Balanced Models}\doublelabel{balanced-models}
 
 \emph{In total we have 8 scalar unknowns and 8 scalar equations, i.e., a
 \textbf{locally} \textbf{balanced model} (and this feature holds for any
-models used for the replaceable component ``t''). }
+models used for the replaceable component \lstinline!t!). }
 
 \emph{Some more analysis reveals that this local set of equations and
 unknowns is structurally non-singular. However, this does not provide
 any guarantees for the global set of equations, and specific
-combinations of models that are ``locally non-singular'' may lead to a
+combinations of models that are locally non-singular may lead to a
 globally non-singular model.}{]}
+% The sentence above looks strange.  Shouldn't it be "globally singular"?
```

Comment:
Yes. it should be, but might be best to have in a separate PR to make it clearer.

Reply:
Sure.  Reported as #2510.